app/lib/system/deploy.rb

```diff
@@ -4,28 +4,44 @@ module System
   module Deploy
     mattr_accessor :info
 
+    DEFAULT_DEPLOY_INFO_PATH = '.deploy_info'
+    private_constant :DEFAULT_DEPLOY_INFO_PATH
+
     # Provides information about the version of the code:
     #   - revision: low-level version, that is used by BugSnag as app version
     #   - release: customer-facing version, that is visible in the admin portal footer
     #   - deployed_at: timestamp of the deployment
-    # The data is taken from the `.deploy_info` file in the root directory, but for SaaS the release
-    # is overridden with VERSION.
+    #   - docs_version: product name and the version that are used as prefix for product documentation path in the
+    #                   Red Hat Customer Portal
+    # The data is taken from the `.deploy_info2` file in the root directory, but for SaaS the release
```

Comment:
`.deploy_info2`?

Reply:
Oops. Good catch!

When testing, I renamed the local file, but the IDE's "refactor" changed it everywhere :sweat_smile: